coherence/session.go

```diff
@@ -349,8 +431,8 @@ func (s *Session) Close() {
 }
 
 func (s *Session) String() string {
-	return fmt.Sprintf("Session{id=%s, closed=%v, caches=%d, maps=%d, options=%v}", s.sessionID.String(), s.closed,
-		len(s.caches), len(s.maps), s.sessOpts)
+	return fmt.Sprintf("Session{id=%s, closed=%v, caches=%d, maps=%d, gRPCv1=%v, options=%v}", s.sessionID.String(), s.closed,
```

Comment:
Maybe gRPCv1 should be protocol-version.  Looking forward to the next rev of the protocol, it doesn't seem to make sense to explicitly list the version in the attribute name.

Reply:
sounds good i will add serverProtocolVersion and rename 
`func (s *Session) IsGrpcV1` to `func (s *Session) IsGrpcV1OrAbove`